docs/features/techdocs/extensions.md

```diff
@@ -0,0 +1,54 @@
+---
+id: extensions
+title: Using TechDocs Extensions
+sidebar_label: Using TechDocs Extensions
+description: How to use the built-in TechDocs extension points
+---
+
+# TechDocs Backend Extensions
+
+The TechDocs backend plugin provides the following extension points:
+
+- `techdocsBuildsExtensionPoint`
+- `techdocsGeneratorExtensionPoint`
+- `techdocsPreparerExtensionPoint`
+- `techdocsPublisherExtensionPoint`
```

Comment:
I think readers would appreciate a sentence explaining what each extension is 🙂 . Could you help us complete the list with a sentence explaining what each extension are made for?